kani-driver/src/call_cbmc.rs

```diff
@@ -40,7 +41,9 @@ impl KaniSession {
             // extra argument
             cmd.arg("--json-ui");
 
+            let now = Instant::now();
             let _cbmc_result = self.run_redirect(cmd, &output_filename)?;
+            let elapsed = now.elapsed().as_secs_f32();
```

Comment:
Why `f32`?

Reply:
What's the alternative you have in mind? Is it `f64`? If so, we probably don't need that much precision, but probably won't hurt much either.